src/lib/ext-listen.js

```diff
@@ -31,3 +31,14 @@ export function openActivityLogPage() {
     url: getActivityLogPageURL(),
   });
 }
+
+export function timeFormat(timestamp) {
+  const date = new Date(timestamp);
+  return `${date.toLocaleTimeString()}`;
+}
+
+export function dateTimeFormat(timestamp) {
+  const date = new Date(timestamp);
+  const month = date.getMonth() + 1;
+  return `${date.getDate()}-${month}-${date.getFullYear()} ${date.toLocaleTimeString()}`;
```

Comment:
`dd-MM-YYYY` looks reasonable to me because the format is the same as where I am from (The Netherlands), but it is not universally true.

Since you're trying to have locale-aware time strings, I suggest to be consistent.

Use the `Intl.DateTimeFormat` API to format the date in a locale-aware way - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat